integration/api/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>beadledom-integration</artifactId>
+    <groupId>com.cerner.beadledom</groupId>
+    <version>3.3-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>beadledom-integration-api</artifactId>
+
+  <dependencies>
```

Comment:
I think there are some used but undeclared dependencies found (you'll be able to find out when you run `mvn dependency:analyze -pl api/`

Reply:
I checked this and other submodules and removed and added some dependencies: https://github.com/cerner/beadledom/pull/149/commits/9ea1691d731343e157a568614742303310dd6651

I'll run this again before I commit as a final sanity check.